about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name = "viewport" content="width=device-width, initial-scale=1">
+    <title>My Recipes</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+
+<body>
+
+    <ul class="menubtn">
+        <li class="close_btn"><img src="images/close-btn.svg" alt="close-btn"></li>
+        <li class="menu-item"><a href="./index.html" class="nav-links">About</a></li>
+        <li class="menu-item"><a href="./index.html#main" class="nav-links">Recipes</a></li>
+        <li class="menu-item"><a href="./about.html#head" class="nav-links">Contact</a></li>
+    </ul>
+    <nav class="navmenu">
+        <div class="nav-title">
```

Comment:
Kindly add a navigation to the home page in the mobile menu so that users can navigate back to the home page if they choose to.

Your current design:
![image](https://user-images.githubusercontent.com/20554583/190858935-2cb1b9b8-f208-4de1-97bb-753939bc022b.png)